Presentation.Web/Tests/it-system/tabs/LocalAdmin.Reference.System.e2e.spec.ts

```diff
@@ -1,86 +1,114 @@
 ﻿import login = require("../../Helpers/LoginHelper");
-import HomePage = require("../../PageObjects/it-system/tabs/ItSystemReference.po");
-import WaitUpTo = require("../../Utility/WaitTimers");
+import ItSystemReferenceHelper = require("../../PageObjects/it-system/tabs/ItSystemReference.po");
 import ReferenceHelper = require("../../Helpers/ReferenceHelper");
-import Constants = require("../../Utility/Constants");
 import TestFixtureWrapper = require("../../Utility/TestFixtureWrapper");
+import ItSystemHelper = require("../../Helpers/SystemCatalogHelper");
 
 var loginHelper = new login();
-var homePage = new HomePage();
-var waitUpTo = new WaitUpTo();
+var itSystemReference = new ItSystemReferenceHelper();
 var refHelper = new ReferenceHelper();
-var consts = new Constants();
 var testFixture = new TestFixtureWrapper();
+var headerButtons = itSystemReference.kendoToolbarWrapper.headerButtons();
+var inputFields = itSystemReference.kendoToolbarWrapper.inputFields();
 
-var headerButtons = homePage.kendoToolbarWrapper.headerButtons();
-var inputFields = homePage.kendoToolbarWrapper.inputFields();
-var colObjects = homePage.kendoToolbarWrapper.columnObjects();
-
-describe("Can add and remove reference",
+describe("Local Admin is able to create,edit and delete",
```

Comment:
Det kommer til at se skørt ud for så bliver testen navngivet: 
**Local Admin is able to create,edit and delete Can add reference to IT-System**
Så skal Describe nok hedde _Local Admin can_ og de enkelte "it" er så "add reference to IT-sytem" osv